src/Manager.cc

```diff
@@ -1106,8 +1111,17 @@ void ManagerPrivate::LoadPlugin(const tinyxml2::XMLElement *_elem)
 
   if (pathToLib.empty())
   {
-    ignerr << "Failed to find the path to library[" << file << "]. "
-      << "Try adding the path to the IGN_LAUNCH_PLUGIN_PATH environment "
+    // TODO(CH3): Deprecated. Remove on ticktock.
+    // This tries to find one more time with IGN_LAUNCH_PLUGIN_PATH instead of
+    // GZ_LAUNCH_PLUGIN_PATH
+    systemPaths.SetPluginPathEnv("IGN_LAUNCH_PLUGIN_PATH");
+    pathToLib = systemPaths.FindSharedLibrary(file);
```

Comment:
It would be nice to print a warning if this is the only way we were able to find a plugin

Reply:
https://github.com/gazebosim/gz-launch/pull/163/commits/223814b51c799b51540b995147dd9f103f3cba8c